src/jest-types.ts

```diff
@@ -15,13 +16,9 @@ export type HasteConfig = {
   providesModuleNodeModules: Array<string>;
 };
 
-export interface PostProcessorOptions {
-  plugins?: any[];
-  presets?: any[];
+
+export interface BabelTransformOptions extends BabelTransformOpts{
   cacheDirectory?: string;
-  filename?: string;
-  retainLines?: boolean;
-  sourceMaps?: string;
 }
```

Comment:
Currently we now inherit from the babel TransformOptions - before we were just typing it ourselves, but what we really have is  standard babel TransformOptions-object with a few extra variables.